MusicStoreTests/ControllersTests/MusicStoreControllerTests.cs

```diff
@@ -10,38 +10,52 @@ namespace MusicStoreTests.ControllersTests
     public class MusicStoreControllerTests
     {
         private Mock<IMusicStoreService> mockMusicStoreService;
+
+        private const int DEFAULT_ID_FOR_ENTITIES = 4;
+        private const int DEFAULT_NEGATIVE_ID_FOR_ENTITIES = -1;
+        private const string EXPECTED_MESSAGE = "Покупка совершена успешно";
+        private readonly Mock<IMapper<MusicStore.DataAccess.BoughtSong, MusicStore.Domain.DataTransfer.BoughtSong>> mockMapBoughtSong;
```

Comment:
You can add one using.
You will not have to write **both** namespaces. Only one